src/cmd/linuxkit/builder.go

```diff
@@ -0,0 +1,84 @@
+package main
```

Comment:
This file should be named `pkg_builder.go`, in line with all of the other files that are subcommands of `pkg`.

Reply:
Done